### ISSUE TITLE
Display Description on Game Page

### ISSUE THREAD

**Issue by nkmcalli** [buttonmen-dev/buttonmen#1508]
I'd like to see the game description on the game page (between the game # and your button columns maybe).  Because:  when I'm playing in tournaments the description shows UBFC or MadRobin, and I like to play those games either first or last.  Having the description on the game page would let me pick those out quickly.

I suppose if a description is very long, this would either look terrible or not work.

An alternative would be to have a flag column, and a way for users to flag specific games themselves, but that seems like a much more complicated feature.  And the description gives more detail.

What do you think?

**Comment by yawetag** [comment]
How about it be on a separate line, spanned across the different columns,
possibly indented to start below "Your Button"? If the game doesn't have a
description, it would just be one line.


Andrew Senger
asenger@gmail.com

**Comment by blackshadowshade** [comment]
Are you aware of the "Next Game" feature? If you click Next Game without playing a turn, then it will take you to the next game, and keep in memory that you wanted to skip playing that game for the time being. This resets when you go back to the Overview page, if I remember correctly.

**Comment by nkmcalli** [comment]
@yawetag: A separate line could work.  I prefer a single line per game though because currently I have  30 games on my overview page, and it would be pretty hard to parse and scroll if split to 60 lines.

**Comment by nkmcalli** [comment]
@blackshadowshade: Yes, sometimes I try the "next game" feature, but I'm trying to get context all at once.  I want to look at the game page and see at a single glance "okay I have 2 UBFC games, and 2 Mad Robin games, and 5 other misc games, now what should I play first".

So as an analogy, when I watch a TV show on Hulu / Netflix / etc, it shows me what my next unwatched episode is, and I can just click to play it, but I don't.  First I click the episode list, so that I can get the context of where I am, and see the previous few episodes as a reminder of where I am.

**Comment by nkmcalli** [comment]
Anyway, this is just a suggestion, just throwing it out there and arguing for it, but I won't be devastated if it doesn't happen.  :)

**Comment by blackshadowshade** [comment]
So, do we have any consensus on the location of this game description text? Anyone feel like mocking up a couple of options using a graphics editor or by tweaking the HTML of an existing Overview page?

**Comment by dwvanstone** [comment]
I don't know if there's room for description on the Overview page, and I think that might be overkill. However, a simple improvement would be to allow a user to mark their game, like one does with email. (With a graphic star or even an asterisk.) That way they know these are special in some way.  Examples below:

Two games are marked:
![screen shot 2015-09-20 at 9 53 36 am](https://cloud.githubusercontent.com/assets/6751265/9981027/f2fe0e3c-5f7d-11e5-871d-81be9934dfd2.png)

When you go into a marked game a menu item allows you to unmark it.
![screen shot 2015-09-20 at 9 55 21 am](https://cloud.githubusercontent.com/assets/6751265/9981029/07bb0668-5f7e-11e5-8268-f51314e15458.png)

And if it's unmarked you can use the menu item to mark it:
![screen shot 2015-09-20 at 9 55 48 am](https://cloud.githubusercontent.com/assets/6751265/9981031/0fcb0376-5f7e-11e5-9a1c-e4dc800b87b3.png)

**Comment by irilyth** [comment]
DVS: That looks a lot like #866, which I think is a great idea.

**Comment by dwvanstone** [comment]
I think the icons in #866 were auto-generated based on the game type. A "mark" (or a "flag" as nkmally wrote) is generated by the user for whatever purposes they have. Let me try mocking up a description in the overview...

**Comment by dwvanstone** [comment]
Here's having a symbol that you can hover over to get the description:

![screen shot 2015-09-20 at 2 31 24 pm](https://cloud.githubusercontent.com/assets/6751265/9982278/5667ebfc-5fa4-11e5-9ebb-3d9e5e350c2c.png)

Here's including the first N characters of the description on the same line as everything else.

![screen shot 2015-09-20 at 2 46 14 pm](https://cloud.githubusercontent.com/assets/6751265/9982335/6ce82638-5fa6-11e5-8914-2028823402e1.png)

I couldn't get a description in the next line to work really well...it's hard to associate the description line directly with the line before it. I can try more later.

**Comment by blackshadowshade** [comment]
I like the idea of having a shortened version of the description. That will definitely show UBFC games, and will encourage people to use descriptions that start with some sort of grouping descriptor. It looks like you're showing roughly 30 characters?

**Comment by dwvanstone** [comment]
Yes, I was basing it on 30 characters. I also forced the column headers to have a line break to shorten the table, but that can be done with other mechanisms.

**Comment by blackshadowshade** [comment]
One obvious place to win quite a lot of horizontal space is the score column.

Also, your change from "Opponent's button" to "Their button" looks like it could make a big difference, even without the line break. What does everyone else think of this wording change?

**Comment by dwvanstone** [comment]
It might be nifty if a shortened description had hover text which showed the complete description.

**Comment by dwvanstone** [comment]
This is my pick for bugfix of the month. I think it'll be helpful for newbies, and it's critical for accepting games to know what the game is about. The "Open Games" shows the description centered on the next line...maybe that's the simplest model to replicate?
